package.json

```diff
@@ -27,6 +27,8 @@
     "@angular/platform-browser-dynamic": "~4.0.3",
     "babel-preset-react": "^6.24.1",
     "core-js": "~2.4.1",
+    "graphiql": "^0.11.11",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
We probably don't need `isomorphic-fetch`.
`fetch` should just work without this.